Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -335,6 +335,13 @@ type AndroidProps = $ReadOnly<{|
    * @platform android
    */
   persistentScrollbar?: ?boolean,
+  /**
+   * The amount of fading.
```

Comment:
Can you add a bit more description here? Assume that what you put here will be the sentence describing what this prop does in our websites api documentation.

Reply:
Thanks for the feedback. I added more description.